src/utils/EVENTS.js

```diff
@@ -1,18 +1,8 @@
 export default {
-  job: {
-    accepted: 'Accepted',
-    started: 'Started',
-    completed: 'Completed',
-    succeeded: 'Succeeded',
-    progress: 'Progress',
-    failed: 'Failed'
-  },
-  client: {
-    accepted: 'JobAccepted',
-    started: 'JobStarted',
-    completed: 'JobCompleted',
-    succeeded: 'JobSucceeded',
-    progress: 'JobProgress',
-    failed: 'JobFailed'
-  }
+  accepted: 'JobAccepted',
```

Comment:
Are we going with "JobAccepted" over "Accepted"? I was thinking the other way aroung

Reply:
According to the GSF docs: 
```
Currently, the server sends the following events:
JobAccepted
JobStarted
JobProgress
JobFailed
JobSucceeded
JobCompleted
```
As an SDK user, I would expect the events I am dealing with to have these same names, not to be stripped of the Job* prefix. Can we reconsider this decision?

Reply:
Also, we have two other objects: JobProgressInfo and JobStartedInfo in the SDK that follow the Job* pattern.

Reply:
I would agree that it seems confusing for the SDK to have different event names than I would see if I used an EventSource client in the browser. Also, if we ever have events for something other than jobs, I think it would be helpful that these events start with "Job", and other events start with something that indicates the source of the event.

Reply:
There was no decision as far as I'm concerned, I just wanted to have the discussion. I think the team has decided on Job*, so that works for me.

Reply:
I'm fine with using the job prefix here.  I have reverted the change.